scintillometry/integration.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 
 import numpy as np
+from astropy.time import Time
```

Comment:
Why is this import here? I don't think `Time` is used directly in `integration.py`